index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Mochiy+Pop+P+One&family=Poppins:wght@300&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="bright-color">
+        <h3 class="purple logo">Hello!</h3>
+        <img id="hamburger-btn" src="Union.png" alt="hamburger" width="40" height="40">
+    </header>
+
+   <main class="bright-color">
+       <section>
+             <h1>I'm Tuğsem <br>Glad to see you</h1>
+             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             <h4 class="purple connect">LET'S CONNECT</h4>
+             <ul class="icons">
+                <li>
+                  <a href="http://twitter.com/TugsemSaranli"><img src="twitter.svg" alt="Twitter"></a>
+                </li>   
+                <li>
+                  <a href="https://linkedin.com/in/tugsem-saranli-5b2a98230/"><img src="Linkedinicon.svg" alt="Linkedin"></a>
+                </li>
+                <li>
+                  <a href="http://github.com/tugsem"><img src="github.svg" alt="Github"></a>
+                </li>   
+             </ul>
+            </section>
```

Comment:
- [ ] It is recommended that you align your items in this section using flex-box as it enhances the responsiveness of your web page. For instance, the icons could be aligned horizontally using the `display: flex` property.